stages/org.osbuild.ostree.deploy.container

```diff
@@ -102,12 +104,53 @@ def ostree_container_deploy(tree, inputs, osname, target_imgref, kopts):
             # consider implicit signature verification type checks, but
             # can't think of a "clean" way to do it yet other than
             # parsing the target-imgref and separating by the ':' character
-            extra_args.append(f'--target-imgref={target_imgref}')
+            # extra_args.append(f'--target-imgref={target_imgref}')
 
             kargs = [f'--karg={v}' for v in kopts]
-
+            
             ostree.cli("container", "image", "deploy",
                        *extra_args, sysroot=tree, *kargs)
+            
+            container_data = subprocess.run(["ostree", "container", "image", "metadata", f"--repo={tree}/ostree/repo", f"{image_source}"],
+					check=True,
+					capture_output=True, 
+					text=True).stdout.rstrip().strip('\"')
+            
+            container_data_config = subprocess.run(["ostree", "container", "image", "metadata", f"--repo={tree}/ostree/repo", f"{image_source}", "--config"],
+					check=True,
+					capture_output=True,
+					text=True).stdout.rstrip().strip('\"')
+            
+            container_data_json = json.loads(container_data)
+            container_data_config_json = json.loads(container_data_config)
+
+            aleph_digest = container_data_json['config']['digest']
+            aleph_ref = f"docker://{target_imgref}@{aleph_digest}"
+            aleph_ostree_version = container_data_config_json['config']['Labels']['version']
+            aleph_ostree_commit = container_data_config_json['config']['Labels']['ostree.commit']
```

Comment:
I'd like to make this optional in the future, please don't hard fail if it's missing.